tasks/main.yml

```diff
@@ -23,7 +23,9 @@
     name: "{{ os_openstacksdk_package_dependencies }}"
     state: present
   become: True
-  when: os_openstacksdk_install_package_dependencies | bool
+  when:
+    - os_openstacksdk_install_package_dependencies | bool
+    - ansible_facts.os_family != "Darwin"
```

Comment:
Seems like we don't need this condition if the list is empty?

Reply:
It tries to execute the task without it:
```TASK [stackhpc.os_openstacksdk : Ensure required packages are installed] *********************************************************************************************************************************************************************
task path: /Users/piotr/repos/forks/ansible-role-os-openstacksdk/tasks/main.yml:21
fatal: [localhost]: FAILED! => {
    "changed": false,
    "msg": "Could not find a module for unknown."
}
```
Possibly it's better to put `os_openstacksdk_install_package_dependencies: False` into Darwin.yml.
```suggestion
```